inception-project/inception
# #838 - Creation of spans in PDF editor

**What's in the PR**
Possibility to create span annotations within PDF editor view.
Closes #838 

**How to test manually**
1. Open a PDF file in PDF editor view
2. Select a layer
3. Select some text in PDF
4. Annotation should appear after a short time
5. Annotation should also appear when changeing to brat editor

**Automatic testing**

**Documentation**
